CHANGELOG.md

```diff
@@ -138,6 +139,16 @@ binary | sha256 hash
 * Scale kube-proxy conntrack limits by cores (new default behavior) ([#28876](https://github.com/kubernetes/kubernetes/pull/28876), [@thockin](https://github.com/thockin))
 * Don't recreate lb cloud resources on kcm restart ([#29082](https://github.com/kubernetes/kubernetes/pull/29082), [@bprashanth](https://github.com/bprashanth))
 
+## Known Issues
+
+There are a number of known issues that have been found and are being worked on.
+Please be aware of them as you test your workloads.
+
+* PVC Volume not detached if pod deleted via namespace deletion ([29051](https://github.com/kubernetes/kubernetes/issues/29051))
```

Comment:
I was sort of following the pattern of the other ones. I'll publish this one like this, since it's long overdue.